proto/tikvpb.proto

```diff
@@ -46,30 +46,32 @@ service Tikv {
     rpc RawDeleteRange(kvrpcpb.RawDeleteRangeRequest) returns (kvrpcpb.RawDeleteRangeResponse) {}
     rpc RawBatchScan(kvrpcpb.RawBatchScanRequest) returns (kvrpcpb.RawBatchScanResponse) {}
 
-    // Store commands (to the whole tikv but not a certain region)
+    // Store commands (sent to a whole TiKV cluster, rather than a certain region).
```

Comment:
I think these interfaces should be for a TiKV instance rather than the whole cluster.

Reply:
AIUI, each message is sent to one instance, but TiDB sends a message to every TiKV instance in the cluster